lua/entities/gmod_tardis/modules/sh_artronenergy.lua

```diff
@@ -0,0 +1,311 @@
+if SERVER then
+
+    local artron_values = {
+        cost_demat = -540,
+        cost_mat = -360,
+        cost_full = -1200,
+        cost_controls = {
+            ["cloak"] = -720,
```

Comment:
Should these controls use energy to turn them off as well as turn them on?

Reply:
It was intended to be like that, so that you can (for instance) be stuck in float mode
I'm unsure about this, but I'd rather give it a go, and only change if it gets lots of negative feedback

Reply:
Okay